static/src/javascripts/projects/common/modules/experiments/tests/facebook-share-params.js

```diff
@@ -0,0 +1,52 @@
+define([
+    'common/utils/config'
+], function (
+    config
+) {
+    return function () {
+        this.id = 'FacebookShareParams';
+        this.start = '2016-06-06';
```

Comment:
Is this the right start date? You'll not be able to opt into the 0% test as this date is in June :)